docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,156 @@
+===================
+Instructors’ Guide:
+===================
+
+Phases of Artemis Online Exam
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- **Exam Creation and Configuration**
+    During the exam creation and configuration, you can create your exam and configure it to fit your needs. Add exercises with different variants, register students, generate student exams. For more information see `Create and Configure Exam`_.
+- **Exam Conduction**
+    The exam conduction starts when the exam becomes visible to the students and ends when the latest working time is over. When the exam conduction begins, you cannot make any changes anymore to the exam configuration or individual student exams. 
+- **Exam Assessment**
+    The assessment begins as soon as the latest student exam working time is over. During this period, your team can assess the submissions of the students and provide results. The testing suites for programming exercises are executed and you can evaluate the quiz exercises automatically, see `Assessing Student Exams and Complaints`_. The exam assessment ends, when the results are published, see `Create and Configure Exam`_.
+- **Exam Student Review**
+    The student review period is set by the exam configuration, see `Create and Configure Exam`_. The students can view their results as soon as they are published, but during the review period then can submit complaints about perceived mistakes made during the exam assessment. A second assessor, other than the original one will have the opportunity to review the complaint and respond to it. The results are then updated automatically. 
+
+
+Accessing the Exam Management Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Log in to Artemis with your account credentials.
+- Head to ``Course Management``.
+- Click on ``Create Exam`` for your course. It will open the *Exam Management Screen*.
+    Here you have access to all the exams of your course. All aspects of the exam are managed from the management screen. 
```

Comment:
I think your "text is bold" issue comes from having a new line and indentation here, which I assume makes rst think the bullet point is some sort of headline. I suggest adding an empty line in-between:
```
- Click on ``Create Exam`` for your course. It will open the *Exam Management Screen*.

    Here you have access to all the exams of your course. All aspects of the exam are managed from the management screen. 
```
That renders for me normally again:
![grafik](https://user-images.githubusercontent.com/72132281/96298032-12fced00-0ff2-11eb-9479-8cdc1503ea4d.png)

Reply:
Yes thank you! Jannis Mainczyk also pointed me to this solution on Slack. I solved this issue